libseccomp/src/arch.rs

```diff
@@ -112,19 +112,11 @@ impl ScmpArch {
     /// This function corresponds to
     /// [`seccomp_arch_native`](https://man7.org/linux/man-pages/man3/seccomp_arch_native.3.html).
     ///
-    /// # Errors
+    /// # Panics
```

Comment:
```rust
fn native() -> Self;
fn try_native() -> Result<Self>;
```

@ManaSugi WDYT

Reply:
@rusty-snake Good. But, after considering about this issue carefully,  I think I don't have to warry about above problem, so it would be fine as it is. Sorry for the confusion.

Reply:
Agree.